.murdock

```diff
@@ -383,8 +383,10 @@ compile() {
         if [ $RUN_TESTS -eq 1 -o "$board" = "native" -o "$EMULATED" = "1" ]; then
             if [ -f "${BINDIR}/.test" ]; then
                 if [ "$board" = "native" -o "${EMULATED}" = "1" ]; then
-                    EMULATE=${EMULATED} BOARD=$board make -C${appdir} test
-                    RES=$?
+                    EMULATE=${EMULATED} BOARD=$board TOOLCHAIN=$toolchain \
+                        dwq-localjob \
+                        --command-override "./.murdock run_test ${appdir} ${board}:${toolchain}" \
+                        -- make -C${appdir} test
```

Comment:
Mind adding a comment here what this does?

Reply:
Very good idea. Done!